protocol/avdl/keybase1/simple_fs.avdl

```diff
@@ -571,4 +572,35 @@ protocol SimpleFS {
   void simpleFSSubscribeNonPath(union{ null, TLFIdentifyBehavior } identifyBehavior, string subscriptionID, SubscriptionTopic topic, int deduplicateIntervalSecond);
 
   void simpleFSUnsubscribe(union{ null, TLFIdentifyBehavior } identifyBehavior, string subscriptionID);
+
+  record DownloadInfo {
+    string downloadID;
+    KBFSPath path;
+    string filename;
+    Time startTime;
+    boolean isRegularDownload;
+  }
+  record DownloadState {
+    string downloadID;
+    double progress;
+    Time endEstimate;
+    string localPath;
+    string error;
+    boolean done;
+    boolean canceled;
+  }
+  record DownloadStatus {
+    array<string> regularDownloadIDs; // the ones showing up in the GUI footer
+    array<DownloadState> states; // all downloads including non-regular ones
+  }
+  // returns download ID. isRegularDownload causes moving to the Downloads
+  // folder. Save on mobile and share to other apps use isRegularDownload=false.
+  string simpleFSStartDownload(KBFSPath path, boolean isRegularDownload);
+  DownloadInfo simpleFSGetDownloadInfo(string downloadID);
+  DownloadStatus simpleFSGetDownloadStatus();
+  void simpleFSCancelDownload(string downloadID);
+  // Remove a download from the download manager. If the download is still
+  // running, this cancels it as well.
+  void simpleFSDismissDownload(string downloadID);
```

Comment:
Seems like this is a required call to free memory in the download manager, right?  If so, that should probably be mentioned in the comment.